manim/mobject/coordinate_systems.py

```diff
@@ -92,28 +92,31 @@ def get_axis_labels(self, x_label_tex="x", y_label_tex="y"):
         return self.axis_labels
 
     def get_graph(self, function, **kwargs):
-        x_min = kwargs.pop("x_min", self.x_min)
-        x_max = kwargs.pop("x_max", self.x_max)
+        x_min = kwargs.pop("x_min", kwargs.pop("t_min",self.x_min))
+        x_max = kwargs.pop("x_max", kwargs.pop("t_max",self.x_max))
+
         graph = ParametricFunction(
             lambda t: self.coords_to_point(t, function(t)),
             t_min=x_min,
             t_max=x_max,
             **kwargs,
         )
-        graph.underlying_function = function
+
+        graph.underlying_function = lambda t: [t,function(t),0]
```

Comment:
This looks like a breaking change that I would like to discuss more. What if `function` already returns a point or array?

I believe the way to go here is to document clearly what the `function` should return.

Reply:
Yes, I agree on documentation. I don't think function can return an array in this case because self.coords_to_point(t,function(t)) would crash if it did so.

`number_plane.get_graph(lambda t: [t,t*t,0],t_min=-2,t_max=2)` throws an error but
`number_plane.get_graph(lambda t: t*t,t_min=-2,t_max=2)` does not.

Maybe a possible merging of get_graph and get_parametric_curve could happen?

Reply:
My logic here is that if this change is really necessary, that would mean that `get_graph` was just broken all this time. Is this the case? Is this change fixing a bug? Or is this case for convenience?

Unfortunately, I am not that familiar with this part of the codebase so I cannot say whether or not get_graph and get_parametric_curve could/should be merged. I'm just trying to prevent breaking changes :slightly_smiling_face: